build.sbt

```diff
@@ -2,65 +2,143 @@ import sbt.ScriptedPlugin
 import sbt.ScriptedPlugin._
 import Dependencies._
 
-version.in(ThisBuild) ~= { old: String =>
-  sys.props.getOrElse("scalafix.version", old.replace('+', '-'))
-}
-name := "scalafixRoot"
-onLoadMessage := s"Welcome to Scalafix ${version.value}"
-noPublish
+lazy val scalaFixedProjects: List[ProjectReference] =
+  List(
+    `scalafix-sbt`,
+    testsInputSbt,
+    testsOutputDotty,
+    testsOutputSbt,
+    website
+  )
 
-lazy val diff = crossProject
-  .in(file("scalafix-diff"))
-  .settings(
-    moduleName := "scalafix-diff",
-    description := "JVM/JS library to build unified diffs."
+lazy val scala212Projects: List[ProjectReference] =
+  List(
+    cli212,
+    core212JS,
+    core212JVM,
+    diff212JS,
+    diff212JVM,
+    reflect212,
+    testkit212,
+    testsInput212,
+    testsOutput212,
+    testsShared212,
+    unit212
   )
-  .jvmSettings(
-    libraryDependencies += googleDiff
+
+lazy val allScala212Projects: List[ProjectReference] =
+  scala212Projects ++ scalaFixedProjects
+
+lazy val scala212ProjectsDependencies: List[ClasspathDep[ProjectReference]] =
+  scala212Projects.map(ClasspathDependency(_, None))
+
+lazy val scala211Projects: List[ProjectReference] =
+  List(
+    cli211,
+    core211JS,
+    core211JVM,
+    diff211JS,
+    diff211JVM,
+    reflect211,
+    testkit211,
+    testsInput211,
+    testsOutput211,
+    testsShared211,
+    unit211
   )
-  .jsSettings(
-    allJSSettings,
-    npmDependencies in Compile += "diff" -> "3.2.0"
+
+lazy val allScala211Projects: List[ProjectReference] =
+  scala211Projects ++ scalaFixedProjects
+
+lazy val scala211ProjectsDependencies: List[ClasspathDep[ProjectReference]] =
+  scala211Projects.map(ClasspathDependency(_, None))
+
+lazy val scalafix = project
+  .in(file("."))
+  .settings(
+    moduleName := "root",
```

Comment:
I avoid "root" because it makes difficult to find the project in the "Open recent" tab in IJ 
![screen shot 2017-11-25 at 09 14 52](https://user-images.githubusercontent.com/1408093/33228686-2ecdb824-d1c1-11e7-904b-b4485823cc04.png)